IssueTracker/DataAccess/Repository/FileRepository.cs

```diff
@@ -0,0 +1,43 @@
+﻿using DataAccess.DbAccess;
+
+namespace DataAccess.Repository;
+public class FileRepository : IFileRepository
+{
+    private readonly ISQLDataAccess _db;
+    public FileRepository(ISQLDataAccess db)
+    {
+        _db = db;
+    }
+    public async Task<string?> AddAsync(Models.File entity)
+    {
+        DateTime upate = DateTime.UtcNow;
+        var result = await _db.SaveDataAndGetIdAsync<dynamic, string>("dbo.spFile_Insert", new
+        {
+            FileId = entity.FileId,
+            Extension = entity.Extension,
+            FileIssueId = entity.FileIssueId,
+            FileCommentId = entity.FileCommentId,
+            Updated = upate
+        });
+        return result;
+    }
+
+    public async Task DeleteAsync(string fileId)
+    {
+        DateTime update = DateTime.UtcNow;
+        await _db.SaveDataAsync("dbo.spFile_Delete", new { FileId = fileId, Updated = update });
+    }
+
+    public async Task<IEnumerable<Models.File>> GetByIssueIdAsync(int issueId)
```

Comment:
Is this needed? You should already have this in the issue details.

Reply:
well, it wouldn't be if Claudiu made the join